### PR TITLE
fix(i18n): 登鹿 for zh-CN & zh-TW

### PR DESCRIPTION
**Pull request by yenche123**
### Description

Is this a pun intentionally made by the developers?

sign_in_elk

---

### What is the purpose of this pull request? 

- [ ] Bug fix
- [ ] New Feature
- [ ] Documentation update
- [X] Translations update
- [ ] Other

### Before submitting the PR, please make sure you do the following

- [X] Read the [Contributing Guidelines](https://github.com/elk-zone/elk/blob/main/CONTRIBUTING.md).
- [X] Check that there isn't already a PR that solves the problem the same way to avoid creating a duplicate.
- [X] Provide related snapshots or videos.
- [X] Provide a description in this PR that addresses **what** the PR is solving, or reference the issue that it solves (e.g. `fixes #123`).

### CODE REVIEW

File: locales/zh-CN.json [hunk 1]
```diff
@@ -61,7 +61,7 @@
     "reset": "重置",
     "save": "保存",
     "save_changes": "保存更改",
-    "sign_in": "登鹿",
+    "sign_in": "登录",
```
Comment (sxzz) [inline locales/zh-CN.json:64]: It's by design.

---

这是故意的。因为叫做「鹿鸣」，所以改「登录」为「登鹿」

Comment (ushuz) [reply locales/zh-CN.json:64]: We could leave some comments there if JSONC is supported.

**Comment by antfu** [comment]
I would suggest we construct a terms table as README in the repo for each language. JSON5 is not a good option as we are using i18n Ally to write back this file (so the comment can't be persisted), and it could be hard to read in a large file. Closing this PR for now.

File: locales/zh-CN.json (hunk 1 above)
Comment (yenche123) [reply locales/zh-CN.json:64]: 好哦，很有趣

**Pull request closed without merging**